vsintegration/src/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -534,15 +534,27 @@ type internal FsiToolWindow() as this =
             executeTextNoHistory null text
         with _ -> ()
 
+    let hide =
+        if sessions.SupportsInteractivePrompt then
+            """#interactiveprompt "hide" """
+        else
+            ""
+
+    let show =
+        if sessions.SupportsInteractivePrompt then
+            """#interactiveprompt "hide" """
+        else
+            ""
+
     let executeInteraction dbgBreak dir filename topLine (text:string) =
         let interaction = $"""
-#interactiveprompt "hide"
+{hide.ToString()}
```

Comment:
Probably don't need a `ToString()` here? It's already a string.

Reply:
Yeah ... I think there is a parser bug, because it's necessary.